src/App.vue

```diff
@@ -175,6 +210,143 @@ li {
   }
 }
 
+// Header Styles
+.site-header {
+  background: url('./assets/images/background-header-mobile.png');
```

Comment:
A JPG file might be a better option for a photography, it generally is smaller than a PNG.